ci/provisioning/post_provision_config_nodes_UBUNTU_20_04.sh

```diff
@@ -53,7 +53,7 @@ post_provision_config_nodes() {
                        lsb-core
 
     # shellcheck disable=2086
-    if [ -n "$INST_RPMS" ] &&
+    if [ -n "$INST_RPMS"] &&
```

Comment:
(lint) The mentioned syntax error was in this if expression. [SC1009]
(lint) Couldn't parse this test expression. Fix to allow more checks. [SC1073]
(lint) Expected this to be an argument to the unary condition. [SC1019]
(lint) You need a space before the ]. [SC1020]
(lint) Missing space before ]. Fix any mentioned problems and try again. [SC1072]